### DISCUSSION TITLE
Edit button padding

### DISCUSSION THREAD

**Discussion by jinsley8** [add2cal/add-to-calendar-button#492] [Q&A]
Is there any way to edit the button padding? I'm using `add-to-calendar-button-react`

This is my button so far:
```
<AddToCalendarButton
    name={`${taskData?.name}`}
    startDate={calStartDate}
    startTime={calStartTime}
    endTime={calEndTime}
    timeZone="America/New_York"
    location={taskData?.address}
    description={taskData?.details}
    options={['Google', 'Microsoft365', 'Outlook.com', 'Apple', 'iCal']}
    buttonsList
    hideTextLabelButton
    buttonStyle="default"
    size="5"
    styleLight="--btn-background:#fff; --btn-background-hover:#fafafa; --btn-border:#fff;--btn-shadow: 'none'; --btn-shadow-hover:'none'; --btn-shadow-active:'none';"
></AddToCalendarButton>
```

**Comment by jekuer** [comment]
Yes. As of v2.5, you can now edit the padding (but we also optimized the default).
Check the styleLight option and check the css of the respective style for available settings.

Maybe also check the PRO offering at https://add-to-calendar-pro.com/.
There, you can create an manage "Styles" for the button. The app holds a nice interface to override any variable and more.
In doubt, you can still use the free OS version (as you do now), but use the app (it's also free in its base version) for better guidance on what you can change.
